.editorconfig

```diff
@@ -15,6 +15,10 @@ insert_final_newline = true
 trim_trailing_whitespace = true
 end_of_line = lf
 
+# generated files.
+[**/generated/**]
+generated_code = true
```

Comment:
stylecop is not analyzing that but IDE (rider) formatter still formats them.

Reply:
Hmmm... I think it might be better to use `.g.cs` (existing convention) for all OAS-generated code and ignore based on that? Feels a bit more robust to me.